_maps/map_files/OmegaStation/OmegaStation.dmm

```diff
@@ -18982,7 +18993,7 @@
 /turf/open/floor/engine,
 /area/engine/engineering)
 "aOa" = (
-/obj/machinery/power/supermatter_shard/crystal/engine,
+/obj/machinery/power/supermatter_shard/crystal/engine/nerfed,
```

Comment:
Instead I'd use power/supermatter_shard/engine with anchored = TRUE since it has the same explosion_power (12).
It generates less power, but from what I remember it's still 200-300kW which is more than enough for Omega.